### DISCUSSION TITLE
how do you set up mainsail with this

### DISCUSSION THREAD

**Discussion by SnipeHype200** [dz0ny/klipper-prusa-mk3s#6] [Q&A] [answered]
hello i am verry confused with how to set this up with fluid so all help is appreciated.

**Comment by SnipeHype200** [comment]
sorry i want to use fluid now

**Comment by dz0ny** [comment, marked as answer]
Start with step 2, https://github.com/dz0ny/klipper-prusa-mk3s#install.
